Adding initial draft of Angular and Auth JS SDK guide (#3173)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/angular/download-sample.md

Update the Okta sample application configuration file (`src/app/okta-config.ts `) in the project root folder. The `okta-config.ts` file contains the configuration values for your Okta app integration. See [Create an app integration](#create-an-app-integration) for details on these values.

```javascript
const oidcConfig = {
  clientId: '${clientId}',
  issuer: 'https://${yourOktaDomain}/oauth2/default',
  scopes: ['openid', 'profile', 'email'],
  pkce: true
};
```

#### Run the sample application

Navigate to the project folder and run the sample application.

```bash
cd okta-angular-oie-sample-quickstart
npm start
```

Open a new browser and navigate to the default local host port (`http://localhost:4200`). Click **Login** and sign in with a user from your Okta org. After a successful authentication, the user's ID token appears on the page.

### Create a new Angular app (optional)

To create an Angular app, open a terminal and install the Angular CLI:

```bash
npm install -g @angular/cli
```

Now, create a new application using the Angular CLI:

```bash
ng new okta-app
```

When asked `Would you like to add Angular routing?`, press "y"

> After all prompts have been answered, the Angular CLI will create a new project in a folder named `okta-app` and installs all required dependencies.

Reviewer: ```suggestion
After all prompts are answered, the Angular CLI creates a new project in a folder named `okta-app` and installs all required dependencies.
```

Update the Okta sample application configuration file (`src/app/okta-config.ts `) in the project root folder. The `okta-config.ts` file contains the configuration values for your Okta app integration. See [Create an app integration](#create-an-app-integration) for details on these values.

```javascript
const oidcConfig = {
  clientId: '${clientId}',
  issuer: 'https://${yourOktaDomain}/oauth2/default',
  scopes: ['openid', 'profile', 'email'],
  pkce: true
};
```

#### Run the sample application

Navigate to the project folder and run the sample application.

```bash
cd okta-angular-oie-sample-quickstart
npm start
```

Open a new browser and navigate to the default local host port (`http://localhost:4200`). Click **Login** and sign in with a user from your Okta org. After a successful authentication, the user's ID token appears on the page.

### Create a new Angular app (optional)

To create an Angular app, open a terminal and install the Angular CLI:

```bash
npm install -g @angular/cli
```

Now, create a new application using the Angular CLI:

```bash
ng new okta-app
```

When asked `Would you like to add Angular routing?`, press "y"

> For this example we are using `CSS` as the style engine. If you would like to use `SCSS` or another style engine, you may need to make a few adjustments to the code snippets shown in this guide.

Reviewer 2: This guide is using SCSS, but there's not really any styles applied so not sure it matters...

Author: I removed this bit. A leftover from the old guide, I believe. Thanks.
